### DISCUSSION TITLE
Don't check "For kids" in YT livestreams

### DISCUSSION THREAD

**Discussion by tacheometry** [supabase/supabase#1733] [Questions] [answered]
Please, don't check "Made for kids" in the video metadata on YouTube. For the "Build in Public" streams, this option has been turned on. We can't comment, use live chat, or use other features.
What YouTube means with "For kids" is actually "Only for kids" - small kids, like the audiences of toy unboxing videos.

**Comment by awalias** [comment]
we're launching a Supabase [Scratch](https://scratch.mit.edu/) integration soon 😆

**Reply by silentworks** [reply]
@awalias let me know when it's ready, my kids build a lot with Scratch. 😁

**Comment by inian** [comment, marked as answer]
The recent streams have comments enabled
